### PR TITLE
[grid-4] Add new session queuer instructions for Grid 4. Add New Session Queuer description for grid components.

### PR DESCRIPTION
**Pull request by pujagani**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Update Grid 4 instructions for running in a fully distributed mode with the New Session Queuer. Add a description of the New Session Queuer and New Session queue functionality in the Grid components.
TODO: Have added the description to just the English language page. Once the description is approved, will add it other language pages.

### Motivation and Context

As a part of https://github.com/SeleniumHQ/selenium/issues/8558, a new component was added to Grid 4 to match the previous functionality of queuing incoming requests. New Session Queuer and New Session Queue are used to queue requests. Distributor picks up the request and creates new sessions. 
To run the Grid 4 in a fully distributed mode, the docs are updates regarding running the session queuer jar and passing the queuer address to distributor and router.

### Types of changes

- [X] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

image

image

### CODE REVIEW

File: docs_source_files/content/grid/grid_4/components_of_a_grid.en.md [hunk 1]
```diff
@@ -56,9 +59,36 @@ forwarding a request to the Node. The Router will ask the Session Map for the No
 associated to a session id. When starting the Grid in its fully distributed mode, the
 Session Map is the first component that should be started.
 
+## New Session Queuer, New Session Queue
+
+The New Session Queuer essentially queues requests. The New Session Queuer is the only
```
Comment (AutomatedTester) [inline docs_source_files/content/grid/grid_4/components_of_a_grid.en.md]:
```suggestion
The New Session Queuer is the only
```

File: docs_source_files/content/grid/grid_4/components_of_a_grid.en.md [hunk 2]
```diff
@@ -56,9 +59,36 @@ forwarding a request to the Node. The Router will ask the Session Map for the No
...
+component which can communicate with the New Session Queue. It handles all queue operations like
+add to front/end, remove request and clear queue. It has configurable parameters for setting
```
Comment (AutomatedTester) [inline docs_source_files/content/grid/grid_4/components_of_a_grid.en.md]:
```suggestion
add to manipulate the queue. It has configurable parameters for setting 
```

File: docs_source_files/content/grid/grid_4/components_of_a_grid.en.md [hunk 3]
```diff
@@ -56,9 +59,36 @@ forwarding a request to the Node. The Router will ask the Session Map for the No
...
+If request times out while retrying or adding to the front of the queue its rejected.
+
+After getting an available slot and session creation, the Distributor passes the new session response 
+to the New Session Queuer via the Event Bus. The New Session Queuer will respond to the client when it
+receives the event.
+
 ## Event Bus
 
-The Event Bus serves as a communication path between the Nodes, Distributor, and Session Map. 
+The Event Bus serves as a communication path between the Nodes, Distributor, New Session Queuer and Session Map.
```
Comment (AutomatedTester) [inline docs_source_files/content/grid/grid_4/components_of_a_grid.en.md]: Always have an oxford comma
```suggestion
The Event Bus serves as a communication path between the Nodes, Distributor, New Session Queuer, and Session Map. 
```

**Review by diemol — APPROVED** [review]
Thank you, @pujagani!

**Pull request merged**